### PR TITLE
[FIX] website_slides: fix a tour that fails randomly

### PR DESCRIPTION
**Pull request by pyduf**
In v17.0, the test TestUiPublisher.test_course_publisher_elearning_manager of website_slides fails randomly at the step: "eLearning: check uploaded pdf presence and perform comparison" with the error: "Cannot read properties of null (reading 'getAttribute')"

How to reproduce it ?
Locally, TestUiPublisher.test_course_publisher_elearning_manager of website_slides runs without errors but when executing it in the browser we get the same error as the runbot systematically:
–	Install website_slides
–	Go to shell mode and execute the setup of the test –	(Backup DB if you want to run the test multiple time) –	Launch in normal mode and connect with demo user –	Go to URL: /web?debug=assets%2Ctests
–	Type “odoo.startTour('course_publisher_standard');” in the console –	An error happens at step “eLearning: check uploaded pdf presence and perform comparison”: Cannot read properties of null (reading 'getAttribute') (to avoid to wait too long, you can shorten the test by keeping mostly “addPdfToSection”)

The problem occurs because the rendering of the iframe o_wslides_iframe_viewer is not always completed when the check is triggered. We solve this problem by waiting for the rendering to finish before performing the test (using "await contains"). For safety, we do the same for another check as this might cause a similar problem in the future.

Although the test has been introduced in 16.2 (see odoo/odoo#117293) but the error happens only in 17.0 (tested only in 16.4 the way described above), we publish only the correction in 17.0 and further to avoid different correction in various version.

Task-3623736

### CODE REVIEW

File: addons/website_slides/static/tests/tours/slides_tour_tools.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /** @odoo-module **/
```
Comment (awa-odoo) [inline addons/website_slides/static/tests/tours/slides_tour_tools.js:1]: Hello @pyduf 

Looking good! Thanks for the quick fix on this one.

Can I just ask you to rephrase that part of the commit message?

> We solve that by adding an
“await contains” on the think we check before the check which ensures that what
we are checking is rendered when checked.

"think" -> "thing" ? Maybe use "step" or something more meaningful?
Also, that's a lot of "check" ^^ Could maybe rephrase a bit to make the sentence clearer.

After that we are good to go :rocket: 

Cheers!

Comment (pyduf) [reply addons/website_slides/static/tests/tours/slides_tour_tools.js:1]: Hello @awa-odoo , 

Yes indeed, thanks for pointing it out. I have rephrased it.

**Comment by awa-odoo** [comment]
@robodoo r+
